src/behaviors.rs

```diff
@@ -13,10 +13,9 @@ pub enum RemoveBehavior {
     /// Use this behavior with caution.
     ///
     /// ```
-    /// use id_tree::Tree;
-    /// use id_tree::Node;
-    /// use id_tree::RemoveBehavior;
-    ///
+    /// # use id_tree::Tree;
+    /// # use id_tree::Node;
+    /// # use id_tree::RemoveBehavior;
     /// let mut tree: Tree<i32> = Tree::new();
```

Comment:
I would like to see this to be consistent with all the other tests. Its weird if you can copy&paste some examples as they are, but others not.
Alternatively you could shorten the import to `use id_Tree::{Tree, Node, RemoveBehavior};` or `use id_tree::*`.